gulp-tasks/utils/constants.js

```diff
@@ -1,7 +1,48 @@
+const uglify = require('rollup-plugin-uglify-es');
+const replace = require('rollup-plugin-replace');
+
 module.exports = {
   // This is a directory that should not be commited
   // to git and will be removed and rebuilt between
   // test runs.
-  BUNDLE_BUILD_DIRNAME: 'bundle-build',
-  ENVIRONMENTS: [undefined, 'production'],
+  PACKAGE_BUILD_DIRNAME: 'build',
+  BROWSER_BUILD_DIRNAME: 'browser-bundles',
+  TEST_BUNDLES_BUILD_DIRNAME: 'bundle-builds',
+
+  // This is the environments that we should use for NODE_ENV.
+  BUILD_TYPES: [undefined, 'production'],
+
+  // Every use of rollup should have minification and the replace
+  // plugin set up and used to ensure as consist set of tests
+  // as possible.
+  getDefaultRollupPlugins: (buildType) => {
```

Comment:
The reason for this is that the rollup plugins that are shared between test bundles and browser bundles *should* be the same. We can inline into the rollup plugins but then we risk the two coming out of sync meaning our bundled tests are now running a more different build than the browser builds.

What do you reckon is best path forward for this?